### PR TITLE
Chapter 3.5 smartmeter webservice slim 447

### PR DESCRIPTION
**Pull request by carlogeenen**
Review documentation for story SLIM-447.

### CODE REVIEW

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://52.16.237.165:80/OSGP/Documentation?pullRequest=36) <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>

File: chapter3/section3.x/ActualMeterReads.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## ActualMeterReads request
```
Comment (robindenadel) [inline chapter3/section3.x/ActualMeterReads.md]: chapter3/section3.x is not correct
the smartmetering documentation is located in chapter3/section5
I think chapter3/smartmetering/ is a better choice

File: chapter3/section5.md [hunk 2]
```diff
@@ -102,3 +102,63 @@ public class ScalerUnitTest {
 ```
```
Comment (robindenadel) [inline chapter3/section5.md]: delete 3.5.1

File: chapter3/section5.md [hunk 3]
```diff
@@ -102,3 +102,63 @@ public class ScalerUnitTest {
...
 #### d. running
 mvn -DskipITs=false verify
+
+### 3.5.2 Webservices
```
Comment (robindenadel) [inline chapter3/section5.md]: In OSLP documentation (http://54.77.62.182/documentation/chapter3/section4.html) the section is called Messages. I think, we should name this section accordingly

**Comment by robindenadel** [comment]
I think chapter3/section2.md should be changed as well.
It now states
3.2.5.2 DLMS

The DLMS implementation is work in progress.

File: chapter3/section5.md (hunk 3 above)
Comment (carlogeenen) [inline chapter3/section5.md]: Done

File: chapter3/section5.md (hunk 2 above)
Comment (carlogeenen) [inline chapter3/section5.md]: Done

**Comment by Sander3003** [comment]
Did you update summary.md as well? This is the index of the gitbook.

File: chapter3/section3.x/ActualMeterReads.md (hunk 1 above)
Comment (carlogeenen) [inline chapter3/section3.x/ActualMeterReads.md]: done

**Pull request merged**